src/MIDebugPackage/OpenFolderSchema.json

```diff
@@ -194,31 +208,48 @@
                   "debuggerPath": {
                     "type": "string",
                     "description": "The full path to the debugger on the target machine, for example /usr/bin/gdb."
-                  },
-                  "pipeEnv": {
+                  },                  "pipeEnv": {
                     "type": "object",
                     "additionalProperties": {
+
                       "type": "string"
                     },
                     "description": "Environment variables passed to the pipe program."
                   },
                   "quoteArgs": {
                     "type": "boolean",
-                    "description": "If individual arguments contain characters (such as spaces or tabs), should it be quoted? Defaults to 'true'. If set to false, the debugger command will no longer be automatically quoted."
+                    "description": "If individual arguments contain characters (such as spaces or tabs), should it be quoted? If 'false', the debugger command will no longer be automatically quoted. \nDefault is 'true'."
+                  }
+                }
+              },
+              "sourceFileMapOptions": {
+                "type": "object",
+                "properties": {
+                  "editorPath": {
+                    "type": "string",
+                    "description": "The location of the source code for the editor to locate."
+                  },
+                  "useForBreakpoints": {
+                    "type": "boolean",
+                    "description": "Use this source mapping when setting breakpoints. If 'false', only the filename and line number is sent. If 'true', only when this mapping applies will the entire path to the file and line number be sent, otherwise just filename and line number is sent. \nDefault is 'true'."
```

Comment:
@gregg-miskelly  I'll take another stab at it. I was trying to avoid the first sentence sounding like a question. 
I can add a '\n' after the first sentence.